index.html

```diff
@@ -8,5 +8,6 @@
   <body>
     <div>Hello World</div>
     <div>Test 2</div>
+    <div>Test 3</div>
```

Comment:
I am suggesting this to make your code better :)
```suggestion
    <div>Test 3, this is a suggestion to your code</div>
```